### PR TITLE
Create AddingIssuesToEpics.md

### PR DESCRIPTION
**Pull request by ACharbonneau**
A first pass at the requested instructions for making issues. Please help me fill in appropriate scopes for the repos I didn't give one for

### CODE REVIEW

File: AddingIssuesToEpics.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+1. Assign someone to do the work using the `Assignees` on the top right
+1. Choose any appropriate labels in the right side panel
+1. Under "Estimate" in the right side panel, assign the task a relative difficulty (low is easy, high is hard, it's entirely subjective, don't put a ton of thought into it)
+1. Under "Epic" in the right side panel, choose the Epic to assign the issue to. You can search by name using the search box that pops up
+1. Click `Submit new issue`
+      
+## Strategy 2 Make the issues first, in bulk
+
+1. In Github, create new issues in the appropriate repos
+      1. Navigate to https://github.com/nih1.cfde
```
Comment (rpwagner) [inline AddingIssuesToEpics.md:25]: Is this a typo? I can't tell if it's a valid link in my browser.

**Comment by rpwagner** [comment]
There are a few places with `https://github.com/nih1.cfde` and I can't tell if those are typos or valid links.

File: AddingIssuesToEpics.md (hunk 1 above)
Comment (ACharbonneau) [reply AddingIssuesToEpics.md:25]: oops

**Comment by ACharbonneau** [comment]
> There are a few places with `https://github.com/nih1.cfde` and I can't tell if those are typos or valid links.

fixed

**Comment by rpwagner** [comment]
Looks good. I'm willing to add some descriptions to the list of repose for what kinds of issues go where.

**Comment by ACharbonneau** [comment]
you should be able to just edit this commit

**Pull request merged**